fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt

```diff
@@ -19,6 +19,7 @@ import com.microsoft.fluentui.tokenized.tabItem.TabItem
 
 data class TabData(
     var title: String,
+    var accessibilityDescription: String? = null,  //Custom announcement for Talkback
```

Comment:
Please move this optional parameter to the end

Reply:
Moved